client/apps/edit/components/content2/section_tool/index.jsx

```diff
@@ -0,0 +1,146 @@
+import React, { Component } from 'react'
+import components from '@artsy/reaction-force/dist/components/publishing/index'
+
+const IconEditEmbed = components.Icon.EditEmbed
+const IconEditImages = components.Icon.EditImages
+const IconEditSection = components.Icon.EditSection
+const IconEditText = components.Icon.EditText
+const IconEditVideo = components.Icon.EditVideo
+
+const IconHeroImage = components.Icon.HeroImage
+const IconHeroVideo = components.Icon.HeroVideo
+
+export default class SectionTool extends Component {
+  constructor (props) {
+    super(props)
+
+    this.type = {displayName: 'SectionTool'}
+    this.state = {
```

Comment:
Not ideal to declare the displayName like this, but it is used by DragDrop component to identify the SectionTool and exclude wrapping it in draggable properties.  Looking into new logic for this but will be updated in a separate PR.